Rakefile

```diff
@@ -4,6 +4,26 @@ require 'date'
 require 'json'
 require 'net/http'
 require 'optparse'
+require "rake/testtask"
+
+# To run all tests:
+#   rake (or) rake test
+#
+# To run one test file:
+#   ruby -Ilib:test test/FILENAME.rb
```

Comment:
Remove this. Better to have just one way to do things.

Reply:
Done.